### PR TITLE
Revert "[IMP] modify menu item for item search extension"

### PR DESCRIPTION
**Pull request by vrenaville**
This reverts commit 78205e0ee4f295b77598c3b72f3766e44cf1bed8.
please see comment :
#82

So I revert the commit that mask the menu Journal Items  and replace it by the entry Journal Items Search All

### CODE REVIEW

**Comment by fclementic2c** [comment]
functional test Ok
:+1:

**Comment by guewen** [comment]
:+1:

**Comment by gurneyalex** [comment]
:+1: looks good to me, would need a review by the participants of the discussion of issue #82: @pedrobaeza, @luc-demeyer

**Comment by pedrobaeza** [comment]
I agree, but as I have said on the issue, we should document it this on module description at least.

**Comment by nbessi** [comment]
The code but it only implement part of the solution proposed in #82 it woud be nice to hide the create button. But I guess it will be harder than a simple `create="false"`.
:+1:  if we open make a separate issue or comment in order not to forget this point.

**Comment by luc-demeyer** [comment]
@nbessi 
The following diff removes the 'create button' : 

--- account_move_line_search_extension.js   2014-10-17 16:28:38.000000000 +0200
+++ account_move_line_search_extension.js.new   2014-11-25 21:47:06.000000000 +0100
@@ -15,6 +15,7 @@
             this.current_partner = null;
             this.current_journal = null;
             this.current_period = null;
- ```
         this.options.addable = false;
   },
  
   start:function(){
  ```

@vrenaville 
can you include this diff and redo the commit ?

**Comment by vrenaville** [comment]
@luc-demeyer , I have juste change according to your comment, but I have reencode the file in unix , because it contain dos CR/LF

**Comment by nbessi** [comment]
:+1:

File: account_move_line_search_extension/static/src/js/account_move_line_search_extension.js [hunk 1]
```diff
@@ -1,93 +1,94 @@
...
+    instance.account_move_line_search_extension.ListSearchView = instance.web.ListView.extend({
+
+        init: function() {
+            this._super.apply(this, arguments);
+            this.journals = [];
+            this.current_account = null;
+            this.current_partner = null;
+            this.current_journal = null;
+            this.current_period = null;
+            this.options.addable = false;
```
Comment (nbessi) [inline account_move_line_search_extension/static/src/js/account_move_line_search_extension.js:18]: Proposed modification is present. It is a shame if the diff is big, but  OK

**Pull request merged**